src/background.js

```diff
@@ -1 +1,163 @@
-/* background.js */
+const activityLogs = [];
+
+const extensionMonitor = {
+  extensionMapList: new Map([]),
+  isMonitorRunning: false,
+
+  async isExtensionPageOpen() {
+    const tab = await browser.tabs.query({
+      url: ['moz-extension://*/activitylog/activitylog.html'],
```

Comment:
Can you use `browser.runtime.getURL()` here, instead of fuzzy-matching using `*`?